src/activity_item_handling.cpp

```diff
@@ -238,28 +238,40 @@ static void put_into_vehicle( Character &c, item_drop_reason reason, const std::
                 it.mod_charges( -charges_added );
                 into_vehicle_count += charges_added;
             }
-            here.add_item_or_charges( where, it );
-            fallen_count += it.count();
+            items_did_not_fit_count += it.count();
+            add_msg( _( "Unable to fit %s in the %2$s's %3$s." ), it.tname(), veh.name, part_name );
+            // if cannot retain in inventory due to being too large/heavy drop onto ground via the drop_on_map after returning
```

Comment:
Does the mentioned `drop_on_map` actually get called as this implies? The only call to `put_into_vehicle` is followed immediately by `return`, which I imagine means it never reaches the `drop_on_map`.

Reply:
Hi yes you are right I had thought the drop_on_map was the function that forced the overflow items to the ground. I've fixed my PR to remedy this.